src/pages/SingleOperatorPage.tsx

```diff
@@ -368,59 +373,32 @@ export const SingleOperatorPage = () => {
                                                 return
                                             }
 
-                                            try {
-                                                // Operator Stake entry is not the same as Sponsorship
-                                                // so we need to do some massaging.
-                                                const sponsorship = {
-                                                    id: element.sponsorshipId,
-                                                    minimumStakingPeriodSeconds:
-                                                        element.minimumStakingPeriodSeconds,
-                                                    stakes: [
-                                                        {
-                                                            amount: fromAtto(
-                                                                element.amountWei,
-                                                            ),
-                                                            operatorId:
-                                                                element.operatorId,
-                                                            joinTimestamp:
-                                                                element.joinTimestamp,
-                                                            metadata: {
-                                                                imageUrl: undefined,
-                                                                imageIpfsCid: undefined,
-                                                                redundancyFactor:
-                                                                    undefined,
-                                                                name: '',
-                                                                description: '',
-                                                            },
-                                                        },
-                                                    ],
-                                                }
+                                            let sponsorship: ParsedSponsorship | null
```

Comment:
Here I decided to nomalize things. I force-fetch the actual sponsorship and use it instead of using the fake hand-made one.